includes/abstracts/class-wc-gateway-mondido-abstract.php

```diff
@@ -802,4 +801,16 @@ public function cancelMondidoSubscription( $subscription_id ) {
 
 		return json_decode( $result['body'], TRUE );
 	}
+
+	private function get_country_alpha2($code)
```

Comment:
What if all crash and burn - do we have any fallback or nice error to throw? :)

Reply:
yeah, we will show a nice wp error message